client/influxdb.go

```diff
@@ -19,7 +19,11 @@ import (
 	"strings"
 	"time"
 
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/csv"
+	fluxClient "github.com/influxdata/influxdb/flux/client"
 	"github.com/influxdata/influxdb/models"
+	"github.com/influxdata/influxdb/services/httpd"
```

Comment:
This dep on the `httpd` package to get the `Response` type may not be a good idea...

Reply:
Could we make a `kit/http` module at the top level, and move `CheckError` over? OSS and IDPE both use that pattern (though not very consistently...)

Reply:
hmm - actually `client` has its own Response struct that matches already, so I'm going to use that instead of doing surgery.